frictionless/eia860.py

```diff
@@ -11,26 +11,10 @@
     "description":
```

Comment:
Given all other data package fields can be built from `pudl.metadata`, what should we do with "name" and "title"? List them here or move them to `pudl.metadata.constants`?

Reply:
If there's metadata that's only ever applicable to the Zenodo archives, and it can't be generated automatically somehow, then I think it should probably live in the `pudl-zenodo-storage` repo. In the case of these titles and names I think we've been doing something pretty procedural though. Like "PUDL Raw {Name of Data Source}" so maybe we only need to store that long name if we aren't already? And probably that would be useful in a documentation and/or Intake Catalog context so it should go in the PUDL repo metadata.